homeassistant/components/ezviz/number.py

```diff
@@ -36,15 +66,18 @@ async def async_setup_entry(
     ]
 
     async_add_entities(
-        EzvizSensor(coordinator, camera, sensor, NUMBER_TYPES)
-        for camera in coordinator.data
-        for sensor, value in coordinator.data[camera].items()
-        if sensor in NUMBER_TYPES.key
-        if value
+        [
+            EzvizSensor(coordinator, camera, NUMBER_TYPES, value)
```

Comment:
We don't need to pass the constant, `NUMBER_TYPES`, around. It's available everywhere in the module below its definition.

Reply:
Done